--- app/build.gradle.kts
+++ app/build.gradle.kts
@@ -37,7 +37,10 @@
       keyPassword = System.getenv("KEY_PASSWORD")
     }
     create("debugConfig") {
-      if (file("${rootDir}/debug.keystore").exists()) { storeFile = file("${rootDir}/debug.keystore") }
+      val debugKeystore = file("${rootDir}/debug.keystore")
+      if (debugKeystore.exists()) {
+          storeFile = debugKeystore
+      }
       storePassword = "android"
       keyAlias = "androiddebugkey"
       keyPassword = "android"
@@ -47,9 +50,15 @@
   buildTypes {
     release {
       isCrunchPngs = false
       isMinifyEnabled = false
       proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
-      signingConfig = signingConfigs.getByName("release")
+      if (signingConfigs.getByName("release").storeFile != null) {
+          signingConfig = signingConfigs.getByName("release")
+      }
     }
-    debug { signingConfig = signingConfigs.getByName("debugConfig") }
+    debug {
+        if (signingConfigs.getByName("debugConfig").storeFile != null) {
+            signingConfig = signingConfigs.getByName("debugConfig")
+        }
+    }
   }
   compileOptions {
     sourceCompatibility = JavaVersion.VERSION_11
